aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -257,22 +353,36 @@ impl<'r, 'l> SessionExt<'r, 'l> {
     ) -> Result<ChangeSetExt, VMStatus> {
         let mut write_set_mut = WriteSetMut::new(Vec::new());
         let mut delta_change_set = DeltaChangeSet::empty();
+        let mut new_slot_metadata: Option<StateValueMetadata> = None;
+        if is_storage_slot_metadata_enabled {
+            if let Some(payer) = new_slot_payer {
+                if let Some(current_time) = current_time {
+                    new_slot_metadata = Some(StateValueMetadata::new(payer, 0, current_time));
```

Comment:
the single payer pays for all slots created in the txn?

Reply:
yes